frontend/src/v5/ui/controls/searchSelect/searchSelect.component.tsx

```diff
@@ -39,7 +39,15 @@ export const SearchSelect = ({ children, ...props }: SelectProps) => {
 		<SearchContextComponent filteringFunction={filterItems} items={children as ReactNode[]}>
 			<SearchContext.Consumer>
 				{ ({ filteredItems }: SearchContextType<typeof MenuItem>) => (
-					<Select {...props} MenuProps={{ disableAutoFocusItem: true }}>
+					<Select
+						{...props}
+						MenuProps={{
+							disableAutoFocusItem: true,
+							PaperProps: {
+								style: { maxHeight: 231 },
```

Comment:
cant we style this in the searchSelectstyles.ts?
```
export const SearchInput = styled(SearchInputBase)`
	margin: 0;
	padding: 2px 12px 12px;
	${Paper} {
	   max-height: 231px;
	}
	
```

Reply:
Unfortunately, this will not work because the `Paper` is not rendered as a descendant of the `Select` component. 
![image](https://user-images.githubusercontent.com/43547703/215818715-7174780d-5bfa-4ba5-b451-95f073ba4817.png)

It would be possible to style it together with the other `v4Overrides.styles` components, but I think that's not the best idea

Reply:
Nope that's for v4 restyling